tests_metricflow/performance/synthetic_manifest_parameter_set.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class SyntheticManifestParameterSet:
+    """Describes how to generate a synthetic manifest for performance testing.
+
+    Goals are:
+    * Allow modeling of similar patterns seen in production manifests.
+    * Make generation straightforward.
+    * Minimize the number of parameters required.
+
+    Notes:
+    * The synthetic manifest groups semantic models into two types - ones containing measures, and others containing dimensions.
```

Comment:
I suppose this still captures the relevant behavior from production models, so it's fine? Is there anything extra we'd able to test by relaxing this and creating "mixed" models?

Reply:
Same question for the other constraints: you can reach all measures via all dimensions etc.

Reply:
So those are simplifications to make it easier to generate semantic models and to make it easier to generate queries for metric. Mixed models would make it mimic production better, but I didn't think the added complexity would flesh out any performance cases that much better.